8.x/core-concepts/cruds.md

```diff
@@ -118,204 +74,234 @@ class Todo extends BaseModel
         "subject"
     ];
 
-    protected static ?string $EXACT_SEARCH_FIELD = "subject";
+    protected static ?string $IMPORTANT_SEARCH_FIELD = "subject";
     protected static ?string $EXACT_SEARCH_ORDER_FIELD = "created_at";
 
     public function getSearchUrl(): string
     {
         return "";
     }
 }
+
 ```
 
-:::tip NOTE
-Note that the `Todo` class extends from the `BaseModel` class.
+:::tip Annotations
+In the code above, the lines number 5 to 11 are called `annotations`. You can study the [annotations](https://docs.larammerce.com/8.x/core-concepts/annotations.html) page to learn more about this topic in the larammerce platform.
 :::
 
-Let's take a deeper look at the written codes in the `Todo` class:
+#### Add route
 
-#### PhpDoc
----
-_PhpDoc_, short for **PhpDocumentor**, is a powerful tool that makes project code documented through specially formatted comments. 
-Also, PHPDoc is an adaptation of Javadoc for the PHP programming language. It is still an informal standard for commenting on PHP code, but it is in the process of being formalized.
+Put the code below in the path `/larammerce/routes/web.php` inside the `admin routes`:
 
-```php
-/**
- * @property integer id
- * @property string subject
- * @property integer status
- * @property Carbon created_at
- * @property Carbon updated_at
- */
+```php{8,9}
+<?php
+...
+//Admin private routes
+    ...
+    //CustomerAddress
+    Route::resource("customer-address", "CustomerAddressController", ["as" => "admin"]);
+
+    //TodoModel
+    Route::resource("todo", "TodoController", ["as" => "admin"]);
+    ...
 ```
 
-:::warning WARNING
-In Larammerce, PhpDoc is an important standard, so every programmer developing a Larammerce project must follow it.
-:::
+#### Add controller
 
-#### $table
----
-The first property that must be write in the todo class is the name of the database table that connects the `todo` class to the `todos` table.
+Create the file `TodoController.php` in the path `/larammerce/app/Http/Controllers/Admin/` and put these codes inside:
 
```

Comment:
add this to the document but change the sentence structure:

The Larammerce project is using Laravel resource controllers. to know more about these controllers and how to use them, refer to this link. --> and lead them to the laravel resource controllers doc.  
here is the link:
https://laravel.com/docs/10.x/controllers#resource-controllers